### PR TITLE
Refactor MC Quiz for JS and CSS

### PR DESCRIPTION
**Pull request by LavaBoy202**
This PR does:
- Refactors JS and CSS quiz components

### CODE REVIEW

File: components/coding/MCQuiz.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        className={`rounded-xl shadow-xl mt-4 p-4 transform transition-all bg-blue-100 ${
+          isShaking ? "animate-shake" : ""
+        } `}
+        onAnimationEnd={() => {
+          setIsShaking(false);
+        }}
+      >
+        <div>
+          {notComplete ? (
+            questionData[index].image ? (
```
Comment (LavaBoy202) [inline components/coding/MCQuiz.tsx:41]: Is using nested Ternary operators good practice?

Comment (vithushan19) [reply components/coding/MCQuiz.tsx:41]: yeah its fine, if its formatted properly

Comment (vithushan19) [inline components/coding/MCQuiz.tsx:41]: I feel like this ternary isn't needed though? Can you just pass down `questionData[index].image` no matter what. And sometimes it'll just be undefined and in that case the image just won't render?

**Review by vithushan19 — APPROVED** [review]
looks good to me. one suggestion about possibly removing one of the ternary's

**Pull request merged**